eng/pipelines/runtime.yml

```diff
@@ -519,7 +519,7 @@ extends:
           jobParameters:
             testScope: innerloop
             nameSuffix: CoreCLR
-            buildArgs: -s clr+libs+host+packs -c $(_BuildConfig) -rc Checked
+            buildArgs: -s clr+libs+host+packs -c $(_BuildConfig) -rc Checked -p:NativeAotSupported=false
```

Comment:
@jkoritzinsky, do we have any precedence where we have multi-steps script? We want to run two steps for freebsd: https://github.com/dotnet/runtime/pull/105004#discussion_r1737107996 (step 2 can be enabled later, but just wanted to highlight that we should have `./build.sh clr.tools+packs -os freebsd` coverage at some point as well)

Reply:
Generally we use `postBuildSteps` to put the pieces together here, possibly using the `global-build-step` template if we need to run the root build script multiple separate times.

Reply:
If you look at the runtime-official pipeline, we have a number of examples.

Reply:
Thank you! Now both steps are in place: https://dev.azure.com/dnceng-public/public/_build/results?buildId=808130&view=logs&j=c2b2f27a-5591-5c35-747e-70b670f3ca20&t=79e2e633-dd91-54d5-0384-9375c8c35f7d :)